src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -21,6 +22,7 @@
 MAX_WORKERS = 32
 SEEN_COUNT_KEY = "seen_count"
 TASK_BUILDS_KEY = "builds"
+ChangedFile = namedtuple("ChangedFile", ["file_name", "repo_name"])
```

Comment:
For your consideration:
```
class ChangedFile(NamedTuple):
    file_name: str
    repo_name: str
```

Reply:
I think a named tuple takes less code. If we add more attributes later, a class would make it easier to manage them, but it's easy to change to a class later.